### PR TITLE
Add software engineer manager job description

### PR DESCRIPTION
**Pull request by nicksnyder**

### CODE REVIEW

File: job-descriptions/software-engineer-manager.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world and about half work from our office in San Francisco.
+
+### Compensation and benefits
+
+We provide competitive pay and equity because we want you to act like a business owner and share in the success of Sourcegraph. We also provide [many benefits](../README.md#benefits) to keep you happy, healthy, and productive.
+
+### Interview process
+
+1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU).
```
Comment (nicksnyder) [inline job-descriptions/software-engineer-manager.md]: TODO: fix link

File: job-descriptions/software-engineer-manager.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+1.  We evaluate relevant skills that you have via a remote interview.
+1.  We schedule a few more hours of technical and non-technical interviews. We're happy to fly you out to our San Francisco office, or conduct the remaining interviews over video chat, whatever works best for you.
+1.  We check your references.
+1.  We make you a job offer.
+
+We also expect you to be interviewing us too, so ask us any questions you have along the way.
+
+If you aren't ready to start interviewing but are interested chat with us about anything, reach out to us [@srcgraph](https://twitter.com/srcgraph) or hiring@sourcegraph.com.
+
+**[Click here to apply](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU)**
```
Comment (nicksnyder) [inline job-descriptions/software-engineer-manager.md]: TODO: fix link

File: job-descriptions/software-engineer-manager.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- Collaborate with the product team and the head of engineering to define and prioritize the work that the team owns.
+- Communicate the goals of the team and the progress toward those goals.
+- Build relationships with the team through 1:1s and team events.
+- Provide career development advice to engineers to help them grow.
+- Hire to increase capacity or fill missing skills on the team.
+
+#### Your qualifications
+
+- You have managed a team before (preferrably at a small startup) and have experience with the resposibilities listed above.
+- You have enough experience as a software engineer to empathize with the needs of the team and our customers.
```
Comment (waynepan) [inline job-descriptions/software-engineer-manager.md]: specific level or software engineer in general? Would be a guide to how technical you want the role to be

File: job-descriptions/software-engineer-manager.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+- **Inclusivity**. To build a product that serves the needs of all developers, we are building a diverse team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+- **Openness**. We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+- **Progress**. We make continuous progress on our product by [releasing a new version of Sourcegraph every month](https://docs.sourcegraph.com/dev/releases). Check out [our product roadmap](https://docs.sourcegraph.com/dev/roadmap) to see what we are working on next. We also hold [regular retrospectives](https://docs.sourcegraph.com/dev/retrospectives) to learn how we can work better as a team.
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+You will build and run one of the engineering teams at Sourcegraph. Here are our current teams and their missions:
```
Comment (mike-saparov) [inline job-descriptions/software-engineer-manager.md]: It's not clear if those teams already have managers and you expect a manager to build another team or if they will take over one of the existing teams.

File: job-descriptions/software-engineer-manager.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+- Collaborate with the product team and the head of engineering to define and prioritize the work that the team owns.
+- Communicate the goals of the team and the progress toward those goals.
+- Build relationships with the team through 1:1s and team events.
+- Provide career development advice to engineers to help them grow.
+- Hire to increase capacity or fill missing skills on the team.
+
+#### Your qualifications
+
+- You have managed a team before (preferrably at a small startup) and have experience with the resposibilities listed above.
```
Comment (mike-saparov) [inline job-descriptions/software-engineer-manager.md]: Do you mean small or quickly growing? :)

File: job-descriptions/software-engineer-manager.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+- You have enough experience as a software engineer to empathize with the needs of the team and our customers.
+- You value quality and have experience making appropriate tradeoffs (e.g. cut scope) to ship quickly and iterate.
+- You are an effective spoken and written communicator.
+- You have have experience interviewing candidates for a software engineer role.
+- You are a humble learner who values continuous growth. You seek out challenge and opportunity.
+
+### Location
+
+We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world and about half work from our office in San Francisco.
```
Comment (mike-saparov) [inline job-descriptions/software-engineer-manager.md]: Any expectations re travel to SFO office if the hire is remote?

File: job-descriptions/software-engineer-manager.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+Our values:
+
+- **Inclusivity**. To build a product that serves the needs of all developers, we are building a diverse team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+- **Openness**. We are an [open company](https://docs.sourcegraph.com/dev/open_source_open_company): our code, our product roadmap, and our company processes are public. This transparency helps us rapidly gather feedback from our customers so we can iterate, learn, and deliver the best product.
+- **Progress**. We make continuous progress on our product by [releasing a new version of Sourcegraph every month](https://docs.sourcegraph.com/dev/releases). Check out [our product roadmap](https://docs.sourcegraph.com/dev/roadmap) to see what we are working on next. We also hold [regular retrospectives](https://docs.sourcegraph.com/dev/retrospectives) to learn how we can work better as a team.
```
Comment (sqs) [inline job-descriptions/software-engineer-manager.md]:
```suggestion
- **Progress**. We make continuous progress on our product by [releasing a new version of Sourcegraph every month](https://about.sourcegraph.com/handbook/engineering/releases). Check out [our product direction](https://about.sourcegraph.com/direction) to see what we are working on next. We also hold [regular retrospectives](https://about.sourcegraph.com/retrospectives) to learn how we can work better as a team.
```

**Comment by nicksnyder** [comment]
Thanks for all the feedback! I am closing since this is old. Will start fresh when the time is right.

**Pull request closed without merging**